test/functional/tests/example/example_test.py

```diff
@@ -16,7 +16,10 @@
 
 
 def setup_module():
-    TestRun.LOGGER.warning("Entering setup method")
+    """
+    Function called by python
```

Comment:
*Python
This function is used nowhere anyway, so IMO can be dropped.

Reply:
I thought so but during debug turned out that it is called by python.py

Reply:
so no - it should not be dropped and that's why the description had been added here.